backend/drm/drm.c

```diff
@@ -50,6 +50,13 @@ bool check_drm_features(struct wlr_drm_backend *drm) {
 		return false;
 	}
 
+	if (drmGetCap(drm->fd, DRM_CAP_CRTC_IN_VBLANK_EVENT, &cap) ||
+		!(cap != 0)) {
+			wlr_log(WLR_ERROR,
```

Comment:
Style: indent the `if` condition continuation lines with two tabs, and the `if` body with one